MovappTests/DictionaryDataStoreTests.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  DictionaryDataStoreTests.swift
+//  MovappTests
+//
+//  Created by Jakub Ruzicka on 25.03.2023.
+//
+
+import XCTest
+@testable import Movapp
+
+final class DictionaryDataStoreTests: XCTestCase {
+
+    func testLoadCsUk() throws {
+        let dataStore = DictionaryDataStore()
+
+        dataStore.load(language: .csUk)
+
+        XCTAssertFalse(dataStore.loading)
+        XCTAssertNil(dataStore.error)
+        XCTAssertNotNil(dataStore.dictionary)
+
+        dataStore.dictionary?.phrases.values.forEach { phrase in
+            if let soundFileName = phrase.source.soundFileName {
+                let sound = NSDataAsset(name: "data/uk-sounds/\(soundFileName)")
```

Comment:
nešlo by to generování cest přesunout do společné logiky? (i u sotires)

Reply:
Teď už je to jednodušší, kdybych měl odstranit kompletně to skládání, bylo by pak potřeba zrefaktorovat celou část ohledně načítání a pouštění zvuků. Do toho se mi teď úplně pouštět nechce.